content/courses/onchain-development/anchor-pdas.md

```diff
@@ -310,42 +312,46 @@ pub struct ReallocExample<'info> {
 }
 
 #[account]
+#[derive(InitSpace)]
 pub struct AccountType {
     pub data: String,
 }
+
+const DISCRIMINATOR: usize = 8;
+const BORSH_LEN: usize = 4;
```

Comment:
One last change - often descriptions or comments give us a good hint for what a better name could be:

>  the `BORSH_LEN` is `4` for the space required to store the length of the
  string
  
Rename `BORSH_LEN` to `STRING_SIZE_SPACE` or similar.

Reply:
done